### DISCUSSION TITLE
Cisco keyword/syntax highlight & session manager

### DISCUSSION THREAD

**Discussion by starrrz** [microsoft/terminal#16963] [Ideas]
Hi,

I work daily connecting via ssh to network devices and currently use MobaXterm and want to move to an open source/free app. 
P.S I know about MRemoteNG - but it does not have keyword highlight.
I have over 400 devices and I need to have a session manager and as well the nice keyword highlight feature.

Is there in plan to implement these features in Windows Terminal?
Is it possible to have keyword/syntax highlight when connecting SSH to Cisco routers/switches, similar with what other tools offer such as Mobaxterm.
![syntax1](https://github.com/microsoft/terminal/assets/93912068/b4e59278-42cd-4a54-a72b-1ffd6ca61809)

![image](https://github.com/microsoft/terminal/assets/93912068/815c1e2a-ff07-4080-9088-426229f868d4)

**Comment by j4james** [comment]
I believe keyword highlighting is tracked in issue #6297, and session management in issue #1280. However, I don't think either feature is likely to be implement soon. They may be better as third-party extensions (issue #4000).

**Reply by starrrz** [reply]
thanks I will look over there!
